webpack.config.js

```diff
@@ -1,22 +1,25 @@
 const path = require('path');
 
 module.exports = {
-  entry: path.resolve(__dirname, 'client', 'index.js'),
+  entry: path.resolve(__dirname, 'client', 'index.jsx'),
```

Comment:
why did you change your extension? Just curious what bugs were here before you change the extension to jsx.

Reply:
The index in Client directory supposed to be .jsx in the first place. eslint is giving me an error when I import .jsx files to a .js file.